packages/legend-application-query/src/stores/LegendQueryApplicationPlugin.ts

```diff
@@ -86,4 +91,9 @@ export abstract class LegendQueryApplicationPlugin extends LegendApplicationPlug
    * Get the list of renderer configurations for existing query editor actions.
    */
   getExtraExistingQueryActionRendererConfiguration?(): ExistingQueryEditorActionRendererConfiguration[];
+
```

Comment:
this is not correct layering-wise, since query loader is now part of query builder (which I agree, it is), we should put this under something like `QueryBuilder_LegendApplicationPlugin_Extension`, look what we did for `DSL_DataSpace_LegendApplicationPlugin_Extension` for example